static/js/nav.js

```diff
@@ -24,6 +24,10 @@ var app = new Vue({
             event.preventDefault();
             await fetch("/api/users/logout", {method : "POST"});
             window.location.href = "/";
```

Comment:
If logout isn't used anymore, get rid of it here

Reply:
fixed